test/tools.test.ts

```diff
@@ -3,222 +3,320 @@
  *  Licensed under the MIT License. See LICENSE file in the project root for license information.
  *-----------------------------------------------------------------------------------------------*/
 
-import { CliExitData, Cli } from '../src/cli';
-import * as assert from 'assert';
+'use strict';
+
+import * as fs from 'fs';
+import * as chai from 'chai';
+import * as path from 'path';
 import * as sinon from 'sinon';
-import { ToolsConfig } from '../src/tools';
-import * as vscode from 'vscode';
+import hasha = require("hasha");
+import { window } from 'vscode';
+import * as fsex from 'fs-extra';
+import * as assert from 'assert';
+import pq = require('proxyquire');
 import * as shelljs from 'shelljs';
-import { Platform } from '../src/util/platform';
+import * as sinonChai from 'sinon-chai';
 import { Archive } from '../src/util/archive';
-import * as path from 'path';
-import * as fs from 'fs';
-import * as fsex from 'fs-extra';
-import hasha = require("hasha");
+import { CliExitData, Cli } from '../src/cli';
+import { Platform } from '../src/util/platform';
+import { ToolsConfig, configData } from '../src/tools';
 
-suite("tools configuration", () => {
-    let sb: sinon.SinonSandbox;
+const expect = chai.expect;
+chai.use(sinonChai);
+
+suite('tools configuration', () => {
+    let sandbox: sinon.SinonSandbox;
     let chmodSyncStub: sinon.SinonStub;
 
     setup(() => {
-        sb = sinon.createSandbox();
-        chmodSyncStub = sb.stub(fs, 'chmodSync');
+        sandbox = sinon.createSandbox();
```

Comment:
```sb``` is abbreviation for ```s(and)b(ox)```, nothing is wrong with it.
Please focus on single task. Do not mix refactoring and your specific task. 

Please restore original name for sand box variable.